jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet

```diff
@@ -234,6 +234,35 @@ function(params) {
     },
   },
 
+  podMonitorKubeProxy: {
```

Comment:
I don't think there is, but asking just to make sure 😬 

If this is being added to the k8s component libsonnet, are there any platform that should not include this podMonitor?

Similar to what happens to some other k8s components, e.g. kube-controller manager in GKE doesn't need a servicemonitor because it is embedded in the apiserver

Reply:
IMHO all platforms should have this disabled by default and enabling should be a conscious administrator choice. This is because by default kube-proxy is not exposing metrics on a non-localhost interface.

Reply:
That is a very good point!

I guess a conditional that can be changed through parameters should be good enough :)